### DISCUSSION TITLE
Enabling/Disabling merge queues through the API

### DISCUSSION THREAD

**Discussion by richard-better** [community/community#50893] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Would it be possible to enable/disable merge queues through the api? It's only possible to edit this through the ui at the moment. 

The branch protection API also doesn't include if you have this enabled: the response is unchanged even if you enable it.

**Comment by dstoyanoff** [comment]
Bumping the topic. This is really a must have

**Comment by StephenWithPH** [comment]
See https://github.com/integrations/terraform-provider-github/issues/1481 for a concrete use case that needs this feature to unblock the use of merge queues by GitHub users.

**Reply by aaron-trout** [reply]
Also safe-settings

**Comment by StephenWithPH** [comment]
This crosses over with https://github.com/orgs/community/discussions/77614
